setup-deb-s3/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
         fi
 
         if ! which lsb_release; then
-          $sudocmd apt-get update
+          $sudocmd apt-get update && true
```

Comment:
I think you meant `|| true`

Also I'd like to see an example where this fails

Reply:
You can see where it fails here: https://github.com/usdot-fhwa-stol/snmp-client/actions/runs/4285308321/jobs/7463443650  It has something to do with using the arm-hf architecture and what the repos return for this.  Strangely it seems the && does work which I was not expecting after your comment so I will try it with || now.

Reply:
Changed it to || and it works as shown here https://github.com/usdot-fhwa-stol/snmp-client/actions/runs/4285355481/jobs/7463552983